enterprise_subsidy/settings/base.py

```diff
@@ -22,6 +22,9 @@ def root(*path_fragments):
     return join(abspath(PROJECT_ROOT), *path_fragments)
 
 
+LMS_URL = os.environ.get('LMS_URL', 'localhost:18000/')
+BULK_ENROLL_REQUEST_TIMEOUT_SECONDS = os.environ.get('BULK_ENROLL_REQUEST_TIMEOUT_SECONDS', 180)
```

Comment:
Should this really be 180 instead of, say, 30?  I think we've talked about avoiding internal timeouts that are longer than load balancer timeouts (the latter of which is currently 60s) because that results in orphaned requests which sometimes do succeed between the 60-180 second mark.